### PR TITLE
ssh_key: Add one new function and a minor fix

### PR DESCRIPTION
**Pull request by dzhengfy**
Add one function to set local host key to remote known_hosts in order to
avoid of prompting when remote host sshes to local host back.

Fix to add parameter 'prompt' for remote_login function

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by will-Do** [comment]
ACK

File: virttest/ssh_key.py [hunk 1]
```diff
@@ -182,3 +183,24 @@ def setup_remote_ssh_key(hostname1, user1, password1,
             session2.close()
         except:
             pass
+
+
+def setup_remote_known_hosts_file(client_ip, server_ip,
+                                  server_user, server_pwd):
+    """
+    Set the host key in local known_hosts file to remote host
```
Comment (waynesun09) [inline virttest/ssh_key.py]: please add param description

Comment (dzhengfy) [inline virttest/ssh_key.py]: @waynesun09 , updated and please review again.

File: virttest/ssh_key.py [hunk 2]
```diff
@@ -182,3 +183,33 @@ def setup_remote_ssh_key(hostname1, user1, password1,
...
+
+    :param client_ip: local host ip address where host key is copied from
+    :type client_ip: str
+    :param server_ip: remote host ip address where host key is copied to
+    :type server_ip: str
+    :param server_user: user to log on remote host
+    :type server_user: str
+    :param server_pwd: password for the user for log on remote host
+    :type server_pwd: str
```
Comment (waynesun09) [inline virttest/ssh_key.py]: returns missing

**Comment by waynesun09** [comment]
ACK

**Pull request merged**